### PR TITLE
Update k8s to v1.20.2

### PR DESCRIPTION
**Pull request by erikwilson**
#### Linked Issues ####
#673

### CODE REVIEW

**Comment by cjellick** [comment]
is there an associated issue for this?

**Comment by briandowns** [comment]
Is there a plan to branch "release-1.19" prior to this merge?

**Comment by erikwilson** [comment]
> is there an associated issue for this?

Added an issue! :)

**Comment by erikwilson** [comment]
> Is there a plan to branch "release-1.19" prior to this merge?

I hadn't planned to create a branch, but we should probably create one pre or post merge.

**Comment by briandowns** [comment]
> > Is there a plan to branch "release-1.19" prior to this merge?
> 
> I hadn't planned to create a branch, but we should probably create one pre or post merge.

Awesome. We have a "release-18" currently so probably want to make a "release-19" if master goes to 1.120. Figured I'd ask. Thanks!

File: Dockerfile [hunk 1]
```diff
@@ -120,15 +120,15 @@ RUN echo ${CACHEBUST}>/dev/null
 RUN CHART_VERSION="v3.13.3"     CHART_FILE=/charts/rke2-canal.yaml             CHART_BOOTSTRAP=true    /charts/build-chart.sh
 RUN CHART_VERSION="1.10.101"    CHART_FILE=/charts/rke2-coredns.yaml           CHART_BOOTSTRAP=true    /charts/build-chart.sh
 RUN CHART_VERSION="1.36.300"    CHART_FILE=/charts/rke2-ingress-nginx.yaml     CHART_BOOTSTRAP=false   /charts/build-chart.sh
-RUN CHART_VERSION="v1.19.7"    CHART_FILE=/charts/rke2-kube-proxy.yaml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
+RUN CHART_VERSION="v1.20.2"     CHART_FILE=/charts/rke2-kube-proxy.yaml        CHART_BOOTSTRAP=true    /charts/build-chart.sh
```
Comment (brandond) [inline Dockerfile:123]: Is there a PR in rke2-charts for this yet?

Comment (erikwilson) [reply Dockerfile:123]: <looking at ci> not yet, didn't realize that was a pre-requirement or that updating k8s in rke2 was so complicated. will work on updating the other stuff also.

Comment (briandowns) [reply Dockerfile:123]: `developer-docs/upgrading_kubernetes.md ` lays it out step-by-step.

Comment (erikwilson) [reply Dockerfile:123]: > `developer-docs/upgrading_kubernetes.md ` lays it out step-by-step.

yes, I am looking at that now, thanks

**Comment by briandowns** [comment]
Should we push a new branch for "release-19" before we merge this upgrade to 20?

**Review by brandond — APPROVED** [review]

**Review by briandowns — APPROVED** [review]

**Pull request merged**